src/bip44.c

```diff
@@ -137,15 +143,15 @@ uint32_t bip44_getColdKeyIndex(const bip44_path_t* pathSpec)
 	return pathSpec->path[BIP44_I_POOL_COLD_KEY];
 }
 
-bool bip44_hasReasonableAccount(const bip44_path_t* pathSpec)
+static bool bip44_hasReasonableAccount(const bip44_path_t* pathSpec)
```

Comment:
Btw, what difference does this `static` make?

Reply:
It limits the scope of the function to a single file.
https://www.scaler.com/topics/static-function-in-c/